packages/at_persistence_secondary_server/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 3.0.43
+- fix: Ignore keys with commit-id's in client persistence
```

Comment:
Can you improve this message please? What's the context within which the keys are being ignored?

Reply:
@gkc: Improved the message and pushed the changes. Can you please check the updated message?